client/client_webhook.py

```diff
@@ -153,39 +154,68 @@ def process_webhook(self):
                                           commit_id)  # The way to know which repo/commit goes to this job request
         if input_format == 'markdown':
             input_format = 'md'
-        job = {
+        payload = {
             "identifier": identifier,
-            "user_token": self.gogs_user_token,
+            "gogs_user_token": self.gogs_user_token,
             "resource_type": manifest.resource['id'],
             "input_format": input_format,
             "output_format": "html",
             "source": source_url,
             "callback": callback_url
         }
 
-        self.logger.info('Telling txManager to setup job.')
-        try:
-            response = TxManager(api_url=self.api_url, cdn_bucket=self.cdn_bucket, logger=self.logger).setup_job(job)
-        except Exception as e:
-            # Fake job so we can still build the build_log.json
-            job = {
-                'job_id': None,
-                'identifier': identifier,
-                'resource_type': manifest.resource['id'],
-                'input_format': input_format,
-                'output_format': 'html',
-                'source': source_url,
-                'callback': callback_url,
-                'message': 'Failed to convert',
-                'status': 'failed',
-                'success': False,
-                'created_at': datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ"),
-                'log': [],
-                'warnings': [],
-                'errors': [e.message]
-            }
+        headers = {"content-type": "application/json"}
```

Comment:
This block of change is to use requests to call tx-manager, rather than going straight to the code. This is so its Stage variables can be passed from the API Gateway.